src/Microsoft.DotNet.ProjectModel/ProjectContext.cs

```diff
@@ -143,15 +140,20 @@ public static IEnumerable<ProjectContext> CreateContextForEachTarget(string proj
 
         public ProjectContext CreateRuntimeContext(IEnumerable<string> runtimeIdentifiers)
         {
-            var context = Create(ProjectFile.ProjectFilePath, TargetFramework, runtimeIdentifiers);
-            if (context.RuntimeIdentifier == null)
+            // Check if there are any runtime targets (i.e. are we portable)
```

Comment:
If they do target `netstandard`, we'll publish just as we do for any other project. With portable apps/redists the TFM is no longer a pivot that defines a runnable application.